publish/dir.props

```diff
@@ -4,8 +4,13 @@
 
   <PropertyGroup>
     <ContainerName Condition="'$(ContainerName)' == ''">dotnet</ContainerName>
+    <ChecksumContainerName Condition="'$(ChecksumContainerName)' == ''">dotnet</ChecksumContainerName>
     <BaseUrl Condition="'$(BaseUrl)' == ''">https://dotnetcli.blob.core.windows.net/</BaseUrl>
     <NuGetPushTimeoutSeconds Condition="'$(NuGetPushTimeoutSeconds)' == ''">3600</NuGetPushTimeoutSeconds>
+    <BinariesRelativePath>Runtime/$(ProductVersion)/$(Channel)</BinariesRelativePath>
```

Comment:
The "channel" should not be in the path.

Also, don't change the binaries and the installer paths here.  @mellinoe is doing that.  I was just talking about the checksum paths being in the new structure.